cmake/developer_package/packaging/nsis.cmake

```diff
@@ -2,40 +2,77 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
-# installation directory
-set(CPACK_PACKAGE_INSTALL_DIRECTORY "Intel")
+macro(ov_nsis_specific_settings)
+    # installation directory
+    set(CPACK_PACKAGE_INSTALL_DIRECTORY "Intel")
 
-# TODO: provide icons
-# set(CPACK_NSIS_MUI_ICON "")
-# set(CPACK_NSIS_MUI_UNIICON "${CPACK_NSIS_MUI_ICON}")
-# set(CPACK_NSIS_MUI_WELCOMEFINISHPAGE_BITMAP "")
-# set(CPACK_NSIS_MUI_UNWELCOMEFINISHPAGE_BITMAP "")
-# set(CPACK_NSIS_MUI_HEADERIMAGE "")
+    # TODO: provide icons
```

Comment:
Maybe we need to create a ticket on this

Reply:
we don't plan to put more resources in NSIS.